src/auction.js

```diff
@@ -353,19 +188,128 @@ export function newAuction({adUnits, adUnitCodes, callback, cbTimeout}) {
     });
 
     _auctionStatus = AUCTION_IN_PROGRESS;
-    adaptermanager.callBids(_adUnits, bidRequests, addBidResponseDecorator(addBidResponse.bind(this)), done.bind(this));
+    adaptermanager.callBids(_adUnits, bidRequests, addBidResponse.bind(this), done.bind(this));
   };
 
   return {
+    addBidReceived,
+    executeCallback,
+    callBids,
+    getTimeout: () => _timeout,
+    getAuctionId: () => _auctionId,
     getAuctionStatus: () => _auctionStatus,
     getAdUnits: () => _adUnits,
     getAdUnitCodes: () => _adUnitCodes,
     getBidRequests: () => _bidderRequests,
     getBidsReceived: () => _bidsReceived,
-    callBids
   }
 }
 
+export const addBidResponse = createHook('asyncSeries', function addBidResponse(adUnitCode, bid) {
+  let auctionInstance = this;
+  let bidRequests = auctionInstance.getBidRequests();
+  let auctionId = auctionInstance.getAuctionId();
+
```

Comment:
What happened to `isValid` and all its code that validated the bid?

Reply:
I moved `isValid` to `bidderFactory`. I think that is the best place to validate bid response.